### PR TITLE
Update psi-ms.obo

### PR DESCRIPTION
**Pull request by nithujohn**
New terms added for software

### CODE REVIEW

**Comment by ypriverol** [comment]
@nithujohn you have to update your branch because you have conflicts.

**Review by edeutsch — CHANGES_REQUESTED** [review]
It looks like Joshua's name was truncated. Also, it would be good to have PMIDs instead of PSI:MS where possible. Perseus should probably be capitalized as a proper name of the tool.

**Comment by ypriverol** [comment]
@nithujohn you have to update again your branch :(

**Review by deeptijk — APPROVED** [review]

File: psi-ms.obo [hunk 1]
```diff
@@ -22674,6 +22674,56 @@ def: "Cloud-based pipeline for quantitative proteomics that enables the reanalys
...
+id: MS:1003429
+name: FragPipe
+def: "A computational platform for analyzing mass spectrometry-based proteomics data." [PSI:MS]
+is_a: MS:1001139 ! quantitation software name
+is_a: MS:1001456 ! analysis software
+
+[Term]
+id: MS:1003430
+name: OpenMS
+def: "A software for LC-MS data management and analysis." [PSI:MS]
```
Comment (ypriverol) [inline psi-ms.obo:22708]: @nithujohn This is the OpenMS citation: `38366242`

File: psi-ms.obo [hunk 2]
```diff
@@ -22674,6 +22674,56 @@ def: "Cloud-based pipeline for quantitative proteomics that enables the reanalys
...
+id: MS:1003431
+name: pLink
+def: "A tool for the analysis of chemically cross-linked proteins using mass spectrometry." [PSI:MS]
+is_a: MS:1001139 ! quantitation software name
+is_a: MS:1001456 ! analysis software
+
+[Term]
+id: MS:1003432
+name: pLink2
+def: "Upgraded version of pLink tool, provides a graphical user interface and faster with newly designed index structure." [PSI:MS]
```
Comment (ypriverol) [inline psi-ms.obo:22722]: @nithujohn Here the citation for pLink `31363125`

**Review by ypriverol — CHANGES_REQUESTED** [review]

File: psi-ms.obo (hunk 1 above)
Comment (nithujohn) [reply psi-ms.obo]: done

File: psi-ms.obo (hunk 2 above)
Comment (nithujohn) [reply psi-ms.obo:22722]: done!

**Review by edeutsch — CHANGES_REQUESTED** [review]
31363125 is the paper for pLink 2. If you look at the abstract, you'll see that they put a space between the pLink and 2.

22772728 is the paper for the original pLink

18327264 would be an appropriate PMID for xQuest

**Comment by ypriverol** [comment]
You are right @edeutsch those are the terms.

**Comment by nithujohn** [comment]
> 31363125 is the paper for pLink 2. If you look at the abstract, you'll see that they put a space between the pLink and 2.
> 
> 22772728 is the paper for the original pLink
> 
> 18327264 would be an appropriate PMID for xQuest

Made the changes, thanks!

**Review by ypriverol — APPROVED** [review]

**Review by edeutsch — APPROVED** [review]

**Pull request merged**